package.json

```
@@ -17,7 +17,8 @@
     "cy:run": "cypress run",
     "cy:run:development": "cypress run --env configFile=development",
     "cy:run:production": "cypress run --env configFile=production",
-    "test:e2e": "start-server-and-test 'yarn start --no-browser' 3000 cy:run:development"
+    "test:e2e": "start-server-and-test 'yarn start --no-browser' 3000 cy:run:development",
+    "test": "react-app-rewired test"
```

Just a very opinionated thing. If this starts to grow it will be nice to have a coverage report. Maybe not important now but something I like to have. We could add some basic flags like this:

```suggestion
    "test": "react-app-rewired test --watchAll=false --coverage --verbose"
```